test/org/sagebionetworks/bridge/cache/CacheProviderTest.java

```diff
@@ -219,7 +211,82 @@ public void addAndRemoveViewFromCacheProvider() throws Exception {
         cachedString = simpleCacheProvider.getString(cacheKey);
         assertNull(cachedString);
     }
+    
+    @Test
+    public void oldUserSessionDeserializedToNewUserSession() {
+        String oldJSON = TestUtils.createJson("{'authenticated':true,'environment':'local',"+
+                "'sessionToken':'ccea2978-f5b9-4377-8194-f887a3e2a19b','internalSessionToken':"+
+                "'4f0937a5-6ebf-451b-84bc-fbf649b9e93c','user':{'id':'6gq4jGXLmAxVbLLmVifKN4',"+
+                "'firstName':'Bridge','lastName':'IT','email':'bridgeit@sagebase.org','studyKey':'api',"+
+                "'sharingScope':'no_sharing','accountCreatedOn':'2016-04-21T16:48:22.386Z',"+
+                "'roles':['admin'],'dataGroups':['group1'],'consentStatuses':{'api':{'name':"+
+                "'Default Consent Group','subpopulationGuid':'api','required':true,'consented':false,"+
+                "'signedMostRecentConsent':true,'type':'ConsentStatus'}},'languages':['en','fr'],"+
+                "'encryptedHealthCode':'TFMkaVFKPD48WissX0bgcD3esBMEshxb3MVgKxHnkXLSEPN4FQMKc01tDbBA"+
+                "VcXx94kMX6ckXVYUZ8wx4iICl08uE+oQr9gorE1hlgAyLAM=','type':'User'},'studyIdentifier':"+
+                "{'identifier':'api','type':'StudyIdentifier'},'type':'UserSession'}");
```

Comment:
Without indentation, this is very difficult to read. Can we use a style like https://github.com/Sage-Bionetworks/BridgePF/blob/develop/test/org/sagebionetworks/bridge/models/upload/UploadSchemaTest.java#L120 ?